packages/snaps-execution-environments/src/webview/ProxyMessageStream.test.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-disable @typescript-eslint/consistent-type-definitions */
```

Comment:
Why?

Reply:
Left over from test. Removed already. Good catch.

Reply:
Seems to still be present